src/Services/NodeBalanceService.ts

```diff
@@ -28,10 +27,14 @@ export class NodeBalanceService {
             return {
                 currentBalance: latestRecord,
                 updatedAt: balances[balances.length - 1].updatedAt,
+                balanceChange: earliestRecord.minus(latestRecord),
                 balanceChangePerc: (latestRecord.minus(earliestRecord))
                     .div(earliestRecord).multipliedBy(100).toFixed(2) + '%'
             }
+        } else {
+            return {
+                currentBalance: 0
```

Comment:
Return other information as well, it's easier for the frontend to handle successful responses the same way. BalanceChangePerc is then 0 and updatedAt can be time now